### DISCUSSION TITLE
symbol lookup error after Update

### DISCUSSION THREAD

**Discussion by Flolaffel** [idaholab/moose#29554] [Q&A General] [answered]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

Hello,

I just updated MOOSE and when I try to run my simulations I now get:

`./simp-opt: symbol lookup error: /home/.local/ruef/miniforge/envs/moose/lib/libmpifort.so.12: undefined symbol: PMPIX_Start_progress_thread`

as the only output. I tried reinstalling moose which didn't work. What to do?

**Comment by Flolaffel** [comment, marked as answer]
It just started working. Unfortunately I don't know what the issue was.

**Reply by GiudGiud** [reply]
probably a left-over compiled object, that eventually got cleaned with a make clobberall or a git clean
